packages/forms/resources/lang/pt_BR/components.php

```diff
@@ -394,7 +394,7 @@
     ],
 
     'tags_input' => [
-        'placeholder' => 'Novo rótulo',
```

Comment:
**Personally, I would keep "Novo rótulo"**.

Reply:
I would go for "Rótulo" ou "Etiqueta", yeah

Reply:
'Tag' is used by Wordpress so I suggested it, 'Etiqueta' sounds good in my opinion...